### PR TITLE
Add pre-PR-submission steps to DEVELOPMENT.md

### PR DESCRIPTION
**Pull request by RochesterinNYC**

### CODE REVIEW

**Comment by segiddins** [comment]
@homu r+

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -33,6 +33,24 @@ Bundler doesn't use a Gemfile to list development dependencies, because when we
...
+
+- Make sure the code formatting and styling adheres to the guidelines. We use Rubocop for this. Lack of formatting adherence will result in automatic Travis build failures.
+
+        $ bin/rubocop -a
+
+- Please run the test suite:
+
+        $ bin/rspec
+
+- If you are unable to run the entire test suite, please run the unit test suite and at least the integration specs related to the command or domain of Bundler that your code changes relate to.
```
Comment (segiddins) [inline DEVELOPMENT.md]: trailing space

Comment (RochesterinNYC) [inline DEVELOPMENT.md]: Hahah, never using the Github web editing UI again, geez.

**Comment by RochesterinNYC** [comment]
Updated.

**Comment by segiddins** [comment]
@homu r+

**Pull request merged**